src/definitions/base.ts

```diff
@@ -0,0 +1,14 @@
+import { base } from 'viem/chains';
+import { EASChainDefinition } from '../core/types';
+
+export const easChainBase: EASChainDefinition = {
+  id: base.id,
+  easGraphqlAPI: 'https://base.easscan.org/graphql',
+  schemaUids: [
+    // VERIFIED_COUNTRY
```

Comment:
Could we add some more info on `VERIFIED COUNTRY` and account is here?

Reply:
what kind of info?